syft/frameworks/torch/hook.py

```diff
@@ -210,6 +214,61 @@ def _hook_pointer_tensor_methods(self):
                 new_method = self.get_hooked_pointer_method(attr)
                 setattr(PointerTensor, attr, new_method)
 
+    def _hook_parameters(self):
+        """
+        This method overrides the torch Parameter class such that
+        it works correctly with our overridden tensor types. The
+        native torch Parameter class kept deleting all of our
+        attributes on our custom tensors, so we wrote our own.
+        """
+
+        class Parameter:
```

Comment:
Is there a way we can code gen this? I would hate to see pysyft break because they changed the source of parameter and we didn't manually copy it

Reply:
Not really - because we don't really use any of their code other than __repr__.